### DISCUSSION TITLE
Как сделать блокировку скрола?

### DISCUSSION THREAD

**Discussion by omqmo** [taiga-family/taiga-ui#4636] [Q&A] [answered]
Если диалоговое окно вписывается во вьюпорт, то основной скрол "как-то" блокируется. Как сделать тоже самое?
То есть, у меня есть слой, который я проэцирую в `ngProjectAs="tuiOverContent"`, но контент под ним продолжает реагировать на колесико мыши. Есть у вас что-то для этого?

**Comment by waterplea** [comment, marked as answer]
Блокировка скролла в диалогах делается через директиву tuiOverscroll="all".
